recipes/sklearn-crfsuite/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2010-2020 Google, Inc. http://angularjs.org
```

Comment:
Is this license correct for this package? I don't see an actual copy of the license upstream in the source repo. Please also submit an issue in the github repo asking them to actually include a license in their source distribution and include a link in the `meta.yaml` to the issue as a reminder for future updates to the feedstock.

Reply:
Removed license file, and have created an issue in the repo. [issue](https://github.com/TeamHG-Memex/sklearn-crfsuite/issues/48)